src/data/MmXsmmFused.hpp

```diff
@@ -180,6 +223,14 @@ class edge::data::MmXsmmFused< float > {
       if( i_group >= m_kernels.size() ) {
         m_descs.resize( i_group+1 );
         m_kernels.resize( i_group+1 );
+        m_kernel_flops.resize( i_group+1 );
+#ifdef _OPENMP
+        for ( int i = 0; i < omp_get_max_threads(); ++i ) {
+          m_kernel_stats[i].resize( i_group+1 );
+        }
+#else
+        m_kernel_stats[0].resize( i_group+1 );
+#endif
```

Comment:
Suggestion: Lets put the ``omp_get_max_threads()`` into the global vars and get rid of the preprocessor stuff, similar to https://github.com/3343/edge/blob/develop/src/parallel/global.h#L49